pkg/actions/lua/formats/delta.go

```diff
@@ -27,6 +27,7 @@ var errUnimplementedProvided = errors.New("unimplemented provider")
 type DeltaClient struct {
 	accessProvider AccessProvider
 	ctx            context.Context
+	m              *blob.URLMux
```

Comment:
- mux
- use non pointer (value) and pass the address as needed

Reply:
done